mod/wikirate/set/right/general_overview.rb

```diff
@@ -0,0 +1,36 @@
+format :html do
+  view :content do |args|
+    handle_edit_general_overview( args ) { super args }
+  end
+
+  view :missing do |args|
+    handle_edit_general_overview( args ) { super args }
+  end
+
+  view :titled_with_edits do |args|
+    handle_edit_general_overview( args ) { super args }
+  end
+
+  view :editor do |args|
+    if claim_name = params[:citable] and claim = Card[claim_name]
```

Comment:
Assignment in condition - you probably meant to use `==`.<br>Use `&&` instead of `and`.